docs/en_us/internal/testing.rst

```diff
@@ -270,6 +270,13 @@ will drop you into pdb on error. This lets you go up and down the stack
 and see what the values of the variables are. Check out `the pdb
 documentation <http://docs.python.org/library/pdb.html>`__
 
+Use this command to put a debugging breakpoint in a test.
```

Comment:
We should point out that if someone actually checks in this code, his/her test will hang (i.e., don't check it in).